pwa/src/components/Login.tsx

```diff
@@ -19,6 +22,10 @@ export default function Login(): ReactElement {
             </div>
         </div>
       </div>
+      <p>Loading API Fetch: {isLoading.toString()}</p>
+      {response && 
```

Comment:
I assume this is effectively:
```
if (response) then {render API fetch content}
```
Is the `&&` syntax the preferred way of doing this in React, or is there an `if statement` syntax?

Reply:
That's the react way  ⚛️